packages/disputer/index.js

```diff
@@ -220,16 +223,16 @@ async function run({
 
 async function Poll(callback) {
   try {
-    if (!process.env.EMP_ADDRESS) {
+    if (!process.env.EMP_ADDRESS && !process.env.FINANCIAL_CONTRACT_ADDRESS) {
       throw new Error(
-        "Bad input arg! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party within your environment variables."
+        "Bad environment variables! Specify an EMP_ADDRESS of FINANCIAL_CONTRACT_ADDRESS for the location of the financial contract the bot is expected to interact with."
       );
     }
 
     // This object is spread when calling the `run` function below. It relies on the object enumeration order and must
     // match the order of parameters defined in the`run` function.
     const executionParameters = {
-      empAddress: process.env.EMP_ADDRESS,
+      financialContractAddress: process.env.EMP_ADDRESS || process.env.FINANCIAL_CONTRACT_ADDRESS,
```

Comment:
in the future we should move over to using `FINANCIAL_CONTRACT_ADDRESS` but it should be backward compatible for now.